pkg/plugins/plugins.go

```diff
@@ -72,6 +72,14 @@ func (pm *PluginManager) Init() error {
 		return errutil.Wrapf(err, "Failed to scan main plugin directory '%s'", plugDir)
 	}
 
+	pm.log.Info("Checking managed plugins")
+	plugDir = path.Join(setting.HomePath, "plugins-managed")
```

Comment:
@torkelo -- thoughts on the actual path name?  maybe `managed-plugins` (I did it as plugins-managed so it shows up next to the plugins source tree)

Reply:
Think I like plugins-bundled or plugins-bundle  more